mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/ElasticSearchIntegrationTest.java

```diff
@@ -227,8 +227,6 @@ void headerSearchShouldIncludeMessageWhenDifferentTypesOnAnIndexedField() throws
                 .build(ClassLoader.getSystemResourceAsStream("eml/mailCustomDateHeader.eml")),
             session).getId();
 
-        elasticSearch.awaitForElasticSearch();
```

Comment:
why?

Reply:
Because we can only await once after all the data to be appended.

Reply:
please put that into the commit description